packages/plugin-core/src/_extension.ts

```diff
@@ -254,13 +254,36 @@ class ExtensionUtils {
       type: workspaceType,
       config: dendronConfig,
     } = workspace;
-    const numNotes = _.size(engine.notes);
+    let numNotes = _.size(engine.notes);
 
     let numNoteRefs = 0;
     let numWikilinks = 0;
     let numBacklinks = 0;
     let numLinkCandidates = 0;
     let numFrontmatterTags = 0;
+    let numTutorialNotes = 0;
+
+    // Note IDs that are part of our tutorial(s). We want to exclude these from
+    // our 'numNotes' telemetry.
+    const tutorialIDs = [
```

Comment:
Not biggie since size is small right now, but I'd use a set here if we want to further optimize